src/app/project/list/list.component.html

```diff
@@ -2,59 +2,113 @@
+                    <div class="form-content">
+                        <mat-form-field class="large-field select-list" *ngIf="lists.length">
+                            <mat-label>
+                                Open from list...
+                            </mat-label>
+                            <mat-select [formControl]="listForm.controls['list']" [(value)]="listIri">
+
+                                <mat-option *ngFor="let list of lists; let first = first" [value]="list.id"
+                                    [matTooltip]="list.comments | kuiStringifyStringLiteral" matTooltipPosition="below">
+                                    {{list.labels | kuiStringifyStringLiteral:'all'}}
+                                </mat-option>
+
+                            </mat-select>
+                        </mat-form-field>
+
+
+                    </div>
+                </form>
+            </span>
+
+            <span class="fill-remaining-space"></span>
+
+            <span class="app-toolbar-action" [class.select-form]="lists.length">
+                <button mat-raised-button color="primary" class="create-list-btn"
+                    (click)="openDialog('createList', 'List', project.id)">
+                    {{lists.length ? 'New list' : 'Create your first list'}}
+                </button>
+            </span>
         </div>
 
-        <!-- list of active lists -->
+    </div>
+
+    <!-- main content: show selected list -->
+    <div *ngIf="list && !loadList" class="list-editor" #listEditor>
+
+        <mat-toolbar class="more-space-bottom">
+            <mat-toolbar-row>
+                <h3 class="mat-title" [matTooltip]="list.labels | kuiStringifyStringLiteral:'all'"
+                    matTooltipPosition="above">{{list.labels | kuiStringifyStringLiteral}}</h3>
+                <span class="fill-remaining-space"></span>
+                <p class="mat-body" [matTooltip]="list.comments | kuiStringifyStringLiteral" matTooltipPosition="above">
+                    {{list.comments | kuiStringifyStringLiteral | kuiTruncate:[64]}}</p>
+                <span class="fill-remaining-space"></span>
+                <button mat-stroked-button
+                    (click)="$event.stopPropagation(); openDialog('editListInfo', list.labels[0].value, list.id)">
+                    Edit info
+                </button>
+            </mat-toolbar-row>
+        </mat-toolbar>
+
+        <app-list-item [list]="list" [parentIri]="list.id" [projectIri]="project.id" [projectcode]="projectcode">
+        </app-list-item>
+
+    </div>
+
+
+    <!-- list of active lists -->
+    <!--
         <mat-accordion class="lists-list">
 
-            <mat-expansion-panel *ngFor="let list of projectLists; let i = index" class="list" (closed)="openPanel = -1" (afterExpand)="openPanel = i" [expanded]="projectLists.length === 1 || openPanel === i">
+            <mat-expansion-panel *ngFor="let list of lists; let i = index" class="list" (closed)="openPanel = -1"
```

Comment:
@kilchenmann this code block can be deleted

Reply:
Done